### ISSUE TITLE
[ERROR] Unable to retrieve server certificates

### ISSUE THREAD

**Issue by pmo-19** [jedisct1/dnscrypt-proxy#316]
I have installed the latest version of dnscrypt-proxy 1.6 from my Launchpad Ubuntu PPA https://launchpad.net/~anton+/+archive/ubuntu/dnscrypt

It was working fine until last Friday, when upon starting my system (Trusty 14.04) I couldn't connect to the Internet. I thought first the server might be down. However trying to connect to any server won't change anything.

Today  I did some investigation and I found that the following command does not get through:

`$ dnscrypt-proxy --local-address=127.0.2.1:53 --resolver-name=dnscrypt.eu-nl --provider-key=67C0:0F2C:21C5:5481:45DD:7CB4:6A27:1AF2:EB96:9931:40A3:09B6:2B8D:1653:1185:9C66 resolver-address=176.56.237.171:443 --test=2`
`[INFO] + DNS Security Extensions are supported`
`[INFO] - [dnscrypt.eu-nl] does not support Namecoin domains`
`[INFO] + Provider supposedly doesn't keep logs`
`[NOTICE] Starting dnscrypt-proxy 1.6.0`
`[INFO] Generating a new session key pair`
`[INFO] Done`
`[ERROR] Unable to retrieve server certificates`
`[INFO] Refetching server certificates`
`[ERROR] Unable to retrieve server certificates`
`[INFO] Refetching server certificates`
`[ERROR] Unable to retrieve server certificates`
`$ `

The shorter cmd. line would yield the same result:

`$ dnscrypt-proxy --local-address=127.0.2.1:53 --resolver-name=dnscrypt.eu-nl --test=2`

Do you have any insight on why suddenly dnscrypt-proxy cannot retrieve any server certificates ?

**Comment by jedisct1** [comment]
Hi,

`dnscrypt.eu-nl` seems to be temporarily down. /cc @simonclausen

You can switch to a different server until it is back up.

**Comment by pmo-19** [comment]
Thanks for the quick answer. I was in fact getting the same error when connecting Friday to Soltysiak @ Poznan, Poland, then I switched to the DNSCrypt.eu Resolver #1 @ The Hague, Holland. I check the ping and thought it was down on both ... Right now the Polish one is working and the French one too. Thanks.

However, dnscrypt-proxy is still not working for me. I can't resolve any names.
`ping: unknown host google.com`

I got back to local-address=127.0.0.2 and that's what I declare as DNS as before when it was working. Still I can't resolve any names and I am at a loss for an explanation.
